__tests__/pet.test.js

```diff
@@ -0,0 +1,125 @@
+const Pet = require('../src/pet');
+
+describe('constructor',() => {
```

Comment:
I dont understand why the tests are structured in the way they are. You have multiple describe blocks with the same name, such as `constructor` on lines 3, 14, 20, 59 & 65, when really all the tests inside these blocks should be grouped under a single describe block:
```
describe("constructor", () => {
    it("returns an object", () => {
        expect(new Pet("Fido")).toBeInstanceOf(Object);
    });
  
    it("sets the name property", () => {
        const petOne = new Pet("Fido");
        expect(petOne.name).toEqual("Fido");
    });
  
    it("has an initial age of 0", () => {
        const petOne = new Pet("Fido");
        expect(petOne.age).toEqual(0);
    });
  
    it("has an initial hunger of 0", () => {
        const petOne = new Pet("Fido");
        expect(petOne.hunger).toEqual(0);
    });
  
    it("has an initial fitness of 10", () => {
        const petOne = new Pet("Fido");
        expect(petOne.fitness).toEqual(10);
    });
});
```
Much cleaner. All the tests there should be consolidated into the correct describe block in this manner.

There should also be a clear line between each of the describe and test blocks like this, so its easier to read.